src/Core/Services/Implementations/UserService.cs

```diff
@@ -1263,21 +1263,29 @@ public async Task<bool> IsLegacyUser(string userId)
         return IsLegacyUser(user);
     }
 
-    public async Task<bool> IsManagedByAnyOrganizationAsync(Guid userId)
+#nullable enable
+    public async Task<bool?> IsManagedByAnyOrganizationAsync(Guid userId)
     {
-        var managingOrganization = await GetOrganizationManagingUserAsync(userId);
-        return managingOrganization != null;
+        var managingOrganizations = await GetOrganizationsManagingUserAsync(userId);
+        return managingOrganizations?.Any();
     }
```

Comment:
Apologies for the late suggestion, but this should return a `bool`, not a `bool?`. The user is either managed or they're not.